pages/en/lb4/Concepts.md

```diff
@@ -18,6 +18,7 @@ LoopBack 4 introduces some new concepts that are important to understand:
 - [**Dependency Injection**](Dependency Injection.html): Technique that separates the construction of dependencies of a class or function from its behavior, to keep the code loosely coupled.
 - [**Repository**](Repositories.html): Type of Service that represents a collection of data within a DataSource.
 - [**Decorator**](Decorators.html): Enables you to annotate or modify your class declarations and members with metadata.
+- [**Server**](Server.html): Placeholder.
```

Comment:
Suggest something like this (adapted from your page below).

Represents implementation for inbound transports and/or protocols such as REST over http, gRPC over http2, graphQL over https. They typically listen for requests on a specific port, handle them, and return appropriate responses. A single application can have multiple server instances listening on different ports and working with different protocols.

Reply:
Excellent suggestion! :100: